packages-exp/auth-exp/src/core/user/user_impl.test.ts

```diff
@@ -105,4 +106,46 @@ describe('core/user/user_impl', () => {
       expect(() => user.delete()).to.throw();
     });
   });
+
+  describe('fromPlainObject', () => {
+    const fpo = UserImpl.fromPlainObject;
```

Comment:
i'm not sure I like these aliases, can we just call the function directly? or import it?

Reply:
Done